### PR TITLE
{Release} Upgrade to Azure CLI 2.32.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,84 @@
...
+**AKS**
+
+* `az aks create`: Add new parameter `--enable-fips-image` to support enabling fips image (#20721)
+* `az aks nodepool add`: Add new parameter `--enable-fips-image` to support enabling fips image (#20721)
+
+**App Service**
+
+* [BREAKING CHANGE] az webapp up: Remove support for the python|3.6 (linux and windows), ruby|2.5 (linux), and php|7.3 (windows) runtimes. Add support for the python|3.9 runtime (linux), php|8.0 (linux), and ruby|2.7 (linux) (#20770)
+* [BREAKING CHANGE] az webapp create: Remove support for the python|3.6 (linux and windows), ruby|2.5 (linux), and php|7.3 (windows) runtimes. Add support for the python|3.9 runtime (linux), php|8.0 (linux), and ruby|2.7 (linux) (#20770)
+* [BREAKING CHANGE] az functionapp create: Remove python 3.6 support (#20770)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:18]:
```suggestion
* [BREAKING CHANGE] `az webapp up`: Remove support for the python|3.6 (linux and windows), ruby|2.5 (linux), and php|7.3 (windows) runtimes. Add support for the python|3.9 runtime (linux), php|8.0 (linux), and ruby|2.7 (linux) (#20770)
* [BREAKING CHANGE] `az webapp create`: Remove support for the python|3.6 (linux and windows), ruby|2.5 (linux), and php|7.3 (windows) runtimes. Add support for the python|3.9 runtime (linux), php|8.0 (linux), and ruby|2.7 (linux) (#20770)
* [BREAKING CHANGE] `az functionapp create`: Remove python 3.6 support (#20770)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,84 @@
...
+* Fix #19550: `az staticwebapp users update`: Allow updating static web app user roles again (#20694)
+* `az logicapp create`: Autogenerate a WS1 App Service Plan when no value for `--plan` or `--consumption-plan-location` is provided (#20678)
+* `az appservice plan create`: Allow creating App Service Plans for Logic Apps (SKUs WS1, WS2, and WS3) (#20678)
+* Fix #20757: `az webapp up`:  Fix list index out of range when no `--plan` argument passed (#20760)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:22]:
```suggestion
* Fix #20757: `az webapp up`: Fix list index out of range when no `--plan` argument passed (#20760)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,84 @@
...
+
+* `az redis identity`: Added support for assigning and modifying Identity. (#20833)
+
+**REST**
+
+* [BREAKING CHANGE] `az rest`: Remove `resourceGroup`, `x509ThumbprintHex` transforms (#19579)
+
+**Role**
+
+* [BREAKING CHANGE] `az ad sp create-for-rbac`: Drop `name` property from the output. Use `appId` instead. (#19808)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:71]:
```suggestion
* [BREAKING CHANGE] `az ad sp create-for-rbac`: Drop `name` property from the output. Use `appId` instead (#19808)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,84 @@
...
+
+**Compute **
+
+* `az vm/vmss create/update`: Expand validate license types for `--license-type` parameter (#20724)
+* `az sig image-definition list-shared`: Add new parameters `--marker` and `--show-next-marker` to support paging (#20618)
+* `az sig image-version list-shared`: Add new parameters `--marker` and `--show-next-marker` to support paging (#20618)
+
+**IoT**
+
+* `az iot hub update`: Adds error handling for file-upload parameters and fixes empty $default storage endpoint errors (#20595)
```
Comment (jsntcy) [inline src/azure-cli/HISTORY.rst:41]:
```suggestion
* `az iot hub update`: Add error handling for file-upload parameters and fixes empty $default storage endpoint errors (#20595)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,84 @@
...
+* Add CI job to build Mariner RPM (#20584)
+* Fix install script on Python 3.10 (#20714)
+
+**Profile**
+
+* `az account show/set`: Add `-n`, `--name` argument (#20777)
+
+**Redis**
+
+* `az redis identity`: Added support for assigning and modifying Identity. (#20833)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:63]:
```suggestion
* `az redis identity`: Add support for assigning and modifying Identity. (#20833)
```

File: src/azure-cli/HISTORY.rst (hunk 4 above)
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:41]:
```suggestion
* `az iot hub update`: Add error handling for file-upload parameters and fixes empty $default storage endpoint errors (#20595)
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,84 @@
...
+* `az iot dps linked-hub create`: Add convenience arguments for linking hubs. (#20682)
+
+**Network**
+
+* Fix #19482: Azure Bastion AAD fix for new CLI core changes (#20648)
+* `az network lb inbound-nat-pool create`: Add new parameter `--backend-pool-name` (#20835)
+
+**Packaging**
+
+* Add CI job to build Mariner RPM (#20584)
```
Comment (jsntcy) [inline src/azure-cli/HISTORY.rst:54]:
```suggestion
```

File: src/azure-cli/HISTORY.rst (hunk 5 above)
Comment (jsntcy) [inline src/azure-cli/HISTORY.rst:63]:
```suggestion
* `az redis identity`: Add support for assigning and modifying Identity. (#20833)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,84 @@
...
+* `az vm/vmss create/update`: Expand validate license types for `--license-type` parameter (#20724)
+* `az sig image-definition list-shared`: Add new parameters `--marker` and `--show-next-marker` to support paging (#20618)
+* `az sig image-version list-shared`: Add new parameters `--marker` and `--show-next-marker` to support paging (#20618)
+
+**IoT**
+
+* `az iot hub update`: Adds error handling for file-upload parameters and fixes empty $default storage endpoint errors (#20595)
+* `az iot central app create`: Add new parameter `--mi-system-assigned` to support creating an app with system-assigned managed identity (#20448)
+* `az iot central app identity show/assign/remove`: Add new commands to manage the system-assigned managed identity to an existing IoT Central app (#20448)
+* `az iot dps access-policy`: Being replaced with `az iot dps policy` (#20682)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:44]:
```suggestion
* `az iot dps access-policy`: Be replaced with `az iot dps policy` (#20682)
```

File: src/azure-cli/HISTORY.rst (hunk 6 above)
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:54]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,84 @@
...
+**Network**
+
+* Fix #19482: Azure Bastion AAD fix for new CLI core changes (#20648)
+* `az network lb inbound-nat-pool create`: Add new parameter `--backend-pool-name` (#20835)
+
+**Packaging**
+
+* Add CI job to build Mariner RPM (#20584)
+* Fix install script on Python 3.10 (#20714)
+
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:56]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,84 @@
...
+* `az sig image-definition list-shared`: Add new parameters `--marker` and `--show-next-marker` to support paging (#20618)
+* `az sig image-version list-shared`: Add new parameters `--marker` and `--show-next-marker` to support paging (#20618)
+
+**IoT**
+
+* `az iot hub update`: Adds error handling for file-upload parameters and fixes empty $default storage endpoint errors (#20595)
+* `az iot central app create`: Add new parameter `--mi-system-assigned` to support creating an app with system-assigned managed identity (#20448)
+* `az iot central app identity show/assign/remove`: Add new commands to manage the system-assigned managed identity to an existing IoT Central app (#20448)
+* `az iot dps access-policy`: Being replaced with `az iot dps policy` (#20682)
+* `az iot dps linked-hub create`: Add convenience arguments for linking hubs. (#20682)
```
Comment (jsntcy) [inline src/azure-cli/HISTORY.rst:45]:
```suggestion
* `az iot dps linked-hub create`: Add convenience arguments for linking hubs (#20682)
```

File: src/azure-cli/HISTORY.rst [hunk 10]
```diff
@@ -3,6 +3,84 @@
...
+**App Service**
+
+* [BREAKING CHANGE] az webapp up: Remove support for the python|3.6 (linux and windows), ruby|2.5 (linux), and php|7.3 (windows) runtimes. Add support for the python|3.9 runtime (linux), php|8.0 (linux), and ruby|2.7 (linux) (#20770)
+* [BREAKING CHANGE] az webapp create: Remove support for the python|3.6 (linux and windows), ruby|2.5 (linux), and php|7.3 (windows) runtimes. Add support for the python|3.9 runtime (linux), php|8.0 (linux), and ruby|2.7 (linux) (#20770)
+* [BREAKING CHANGE] az functionapp create: Remove python 3.6 support (#20770)
+* Fix #19550: `az staticwebapp users update`: Allow updating static web app user roles again (#20694)
+* `az logicapp create`: Autogenerate a WS1 App Service Plan when no value for `--plan` or `--consumption-plan-location` is provided (#20678)
+* `az appservice plan create`: Allow creating App Service Plans for Logic Apps (SKUs WS1, WS2, and WS3) (#20678)
+* Fix #20757: `az webapp up`:  Fix list index out of range when no `--plan` argument passed (#20760)
+* Fix  #18652: `az webapp up`: Search for *.csproj in child directories (#20738)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:23]:
```suggestion
* Fix #18652: `az webapp up`: Search for \*.csproj in child directories (#20738)
```

**Review by zhoxing-ms — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Comment by yonzhan** [comment]
Release

**Pull request merged**